lsp/build.gradle

```diff
@@ -10,13 +10,6 @@ dependencies {
         exclude group: 'org.eclipse.platform', module: 'org.eclipse.swt.*'
         exclude group: 'org.eclipse.platform', module: 'org.eclipse.swt'
     }
-
-    // This dependency ensures correct animations and bookkeeping during updates
-    // See https://github.com/lf-lang/vscode-lingua-franca/issues/103#issuecomment-1731023470
-    implementation ("de.cau.cs.kieler.klighd:de.cau.cs.kieler.klighd.incremental:$klighdVersion") {
```

Comment:
@soerendomroes: are you sure his is safe to remove?

Reply:
Since you commented on https://github.com/lf-lang/vscode-lingua-franca/issues/103#issuecomment-1731023470 in the past, I assume that you concluded it was safe to remove. Let me resolve this for now so that we can merge.

Reply:
It was safe to remove, see my last comment's number 3.